pydantic_ai_slim/pydantic_ai/messages.py

```diff
@@ -176,28 +174,30 @@ def has_content(self) -> bool:
             return bool(self.args.args_json)
 
 
+ModelResponseContentItem = Union[TextItem, ToolCall]
+
+
 @dataclass
-class ModelStructuredResponse:
-    """A structured response from a model.
+class ModelResponse:
+    """A response from a model."""
 
-    This is used either to call a tool or to return a structured response from an agent run.
-    """
+    items: list[ModelResponseContentItem]
```

Comment:
Happy to rename this field, I just didn't use `content` initially because anthropic does and I wanted to avoid confusion for myself. Happy to use that name or something else. The only thing I don't like about `content` is it doesn't lend itself to be a part of the names of the items. (I.e., I would like `blocks: list[TextBlock, ToolCallBlock, ...]` or `items: list[TextItem, ToolCallItem, ...]` more than `content: list[TextBlock, ToolCall, ...]` but it's not a strong opinion.)

Reply:
decision: rename to parts